packages/fxa-auth-server/package.json

```diff
@@ -14,25 +14,25 @@
     "audit": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
     "lint": "eslint .",
     "postinstall": "grunt merge-ftl && ../../_scripts/clone-l10n.sh fxa-auth-server",
-    "start": "scripts/start-local.sh",
+    "start": "yarn emails-scss && scripts/start-local.sh",
```

Comment:
@vbudhram This is what fixed all of those functional/content-server test failures